examples/lua_REPL/Makefile

```diff
@@ -0,0 +1,53 @@
+APPLICATION = lua_repl
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_WHITELIST := frdm-k64f iotlab-m3 mulle native nrf52840dk nucleo-f411re \
+				   stm32f4discovery
```

Comment:
Those are boards with enough flash to run Lua. Maybe other boards can run Lua too, but I don't know.

Reply:
Could you at least add this information as a comment here. If you like, maybe you could quickly find all boards that have enough flash and ram to fit the example here.

As this is now, I don't think it is clear for an end user why these specific boards are whitelisted and when a board qualifies

Reply:
Now I'm thinking on whitelisting only the boards that I can try myself. I could run the CI on all boards and then blacklist all that fail because of low memory, but that's not very robust: a later change to riot may cause the build to fail. How should I proceed?

Reply:
Check back with @cladmi .